docs/howto/django.rst

```diff
@@ -5,36 +5,99 @@ Install procrastinate with:
+    $ (venv) pip install procrastinate 'procrastinate[django]'
 
-This tells pip to consider the extra dependencies from the group of dependencies named
-``django``. For now, this group only contains Django itself, which you likely already
-have in your project's dependencies. So why bother?
+This tells pip to install procrastinate but also to consider the extra
+dependencies from the group of dependencies named ``django``. For now, this
+group only contains Django itself, which you likely already have in your
+project's dependencies. So why bother?
 
-Specifying your dependency to the "``django`` extras" will ensure that your Django
-version and the one we support stay in sync through time (for now, we support every
-version, but if we learn of strong incompatibilities, we'll update the lib: we're
-considering every version is compatible until proven otherwise). Also, while this is not
-the case today, if our Django integration ever requires other third-party packages, they
-will be added here.
+Specifying your dependency to the "``django`` extras" will ensure that your
+Django version and the one we support stay in sync through time (for now, we
+support every version, but if we learn of strong incompatibilities, we'll
+update the lib: we're considering every version is compatible until proven
+otherwise). Also, while this is not the case today, if our Django integration
+ever requires other third-party packages, they will be added here.
 
-Add Procrastinate in your ``INSTALLED_APPS``::
+Add procrastinate Django app to your ``INSTALLED_APPS``:
+
+.. code-block:: python
 
     INSTALLED_APPS = [
         ...
         "procrastinate.contrib.django",
     ]
 
-Configure your procrastinate app from your Django settings::
 
-    import procrastinate
+After that, you need to create a new directory to contain the procrastinate
+app, where you will define your tasks. This app will run independently of
+Django but will be able to access the whole Django ecosystem (it's not a Django
+app, nor will be inside any of your existing Django apps). Create a new
+directory ``tasks`` and fill the ``tasks/__init__.py`` with:
+
+.. code-block:: python
+
+    import django
+    django.setup()  # Setup Django inside the worker so you can import/use ORM etc.
+
+    from procrastinate import App, AiopgConnector
     from procrastinate.contrib.django import connector_params
 
-    app = procrastinate.App(
-        connector=procrastinate.Psycopg2Connector(**connector_params())
-    )
+    from myapp.models import MyModel
+
+
+    app = App(connector=AiopgConnector(**connector_params()))
+    app.open()
+
+    @app.task(name="mytask")
+    def mytask(obj_pk):
+        print(f"Executing mytask for object {obj_pk}...")
+        obj = MyModel.objects.get(pj=obj_pk)
+        ...
+
+To run the procrastinate worker properly, you need to set
+``DJANGO_SETTINGS_MODULE`` to your project's settings module and point to the
+``tasks`` app you just created:
```

Comment:
Hm, I'm sure we can change the lib to do something more straightforward than this (but it's a good idea to document what we currently have, because we don't know when we'll change it)

Reply:
Sure! This was my first PR but I'd like to contribute more.
I was thinking in creating a `procrastinate` management command (+ a settings var `PROCRASTINATE_APP` with the dotted app path) that deals with everything, so the user just run `python manage.py procrastinate`.